staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -715,6 +720,10 @@ func (c *Cacher) GetList(ctx context.Context, key string, opts storage.ListOptio
 			listVal.Set(reflect.Append(listVal, reflect.ValueOf(elem.Object).Elem()))
 		}
 	}
+	if listVal.IsNil() {
+		// Ensure that we never return a nil Items pointer in the result for consistency.
```

Comment:
consistency with what? I couldn't find a spot in store#GetList where we were doing a similar `nil → []` transition

Reply:
This isn't explicit - see my response to your comment for etcd3 changes.